modules/pubmaticBidAdapter.js

```diff
@@ -704,22 +710,27 @@ function _handleEids(payload, validBidRequests) {
   }
 }
 
-function _checkMediaType(adm, newBid) {
-  // Create a regex here to check the strings
-  var admStr = '';
-  var videoRegex = new RegExp(/VAST\s+version/);
-  if (adm.indexOf('span class="PubAPIAd"') >= 0) {
-    newBid.mediaType = BANNER;
-  } else if (videoRegex.test(adm)) {
-    newBid.mediaType = VIDEO;
+function _checkMediaType(bid, newBid) {
+  if (bid.ext && bid.ext.bidType && Object.keys(MEDIATYPE).indexOf(bid.ext.bidType.toString())) {
```

Comment:
Hi Shashank,

As discussed in code review, pls check if we need to add check for Object.keys(MEDIATYPE).indexOf(bid.ext.bidType.toString()) > -1

Thanks,
Prakhar Jain